<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTableDialog</class>
 <widget class="QDialog" name="AddTableDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add table - Manuwrite</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="AutogenIdentifierCheckbox">
     <property name="text">
      <string>Generate identifier automatically</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="AutonumberCheckbox">
     <property name="text">
      <string>Add table number automatically</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="MainIdentifierLayout">
     <item>
      <layout class="QVBoxLayout" name="LabelsVerticalLayout">
       <item>
        <widget class="QLabel" name="IdentifierLabel">
         <property name="text">
          <string>Identifier</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="CaptionLabel">
         <property name="text">
          <string>Caption</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="InputsVerticalLayout">
       <item>
        <widget class="QLineEdit" name="IdentifierLineEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="CaptionLineEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="LoadFromFileLabel">
       <property name="text">
        <string>Load from file:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="LoadFromFileButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="TableTypeLayout">
        <item>
         <widget class="QLabel" name="TableTypeLabel">
          <property name="text">
           <string>Table type:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="TableTypeCombobox">
          <property name="currentText">
           <string>Simple table</string>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <item>
           <property name="text">
            <string>Simple table</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="RowsLabel">
          <property name="text">
           <string>Rows:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="RowsSpinBox"/>
        </item>
        <item>
         <widget class="QLabel" name="ColumnsLabel">
          <property name="text">
           <string>Columns:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="ColumnsSpinBox"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="TablePreviewWidget"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddTableDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddTableDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
